src/ducks/triggers/index.js

```diff
@@ -46,14 +46,44 @@ export const deleteTrigger = trigger =>
 export const launchTrigger = trigger => fromCozyClient.launchTrigger(trigger)
 
 // Helpers
+export const buildTriggerFrequencyOptions = (konnector, options) => {
+  const { frequency } = konnector
+  const { day, hours, minutes } = options
+
+  const frequencyOptions = {
+    frequency:
+      frequency && ['weekly', 'daily'].includes(frequency)
```

Comment:
Maybe use a constant to store this array? And have the 'authorized' value stored somewhere

Reply:
Done 👍